java/src/main/resources/templates/activities.ftl

```diff
@@ -0,0 +1,36 @@
+
+<#include "standardPage.ftl" />
+
+<@standardPage title="Api Java Example">
+	<h1>Partner API Demo Site</h1>
+	<h2>Welcome to the Climate Partner Demo App.</h2>
+	<table>
+		<tr>
+		  	<td>ID</td>
```

Comment:
Nit: Use `Id` to be consistent with `Field Ids` on line 15 ?

Reply:
fixed